lerna.json

```diff
@@ -2,5 +2,10 @@
   "packages": [
     "packages/*"
   ],
-  "version": "0.0.0"
+  "version": "0.0.0",
+  "command": {
+    "bootstrap": {
+      "ignore": "ethereumjs-block"
```

Comment:
This is important: it tells Lerna to ignore `ethereumjs-block` from `lerna bootstrap` commands. It will still be used on `lerna run` and `lerna exec` for the time being.